dtfs-central-api/src/services/repositories/utilisation-reports-repo.js

```diff
@@ -19,7 +23,7 @@ const saveUtilisationReportDetails = async (month, year, csvFilePath, uploadedBy
     month: Number(month),
     year: Number(year),
     dateUploaded: new Date(),
-    path: csvFilePath,
```

Comment:
I _think_ this `path` value was what the `previous-reports.njk` template was previously using for the link?